homeassistant/helpers/config_entry_flow.py

```diff
@@ -7,6 +7,13 @@
 
 from homeassistant import config_entries
 from homeassistant.components import onboarding
+from homeassistant.components.cloud import (
```

Comment:
Do we need a local import to not depend on the cloud in this helper unless cloud is already setup, or is it safe to import these functions?